presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -577,32 +578,99 @@ output expressions:
 
 Each expression may be composed of output columns or it may be an ordinal
 number selecting an output column by position (starting at one). The
-``ORDER BY`` clause is evaluated as the last step of a query after any
-``GROUP BY`` or ``HAVING`` clause. The default null ordering is ``NULLS LAST``,
-regardless of the ordering direction.
+``ORDER BY`` clause is evaluated after any ``GROUP BY`` or ``HAVING`` clause
+and before any ``OFFSET``, ``LIMIT`` or ``FETCH FIRST`` clause.
```

Comment:
This should just be `FETCH` (or we'd need to say `FETCH FIRST` and `FETCH NEXT`)

Reply:
In spec, this clause is referred to as "FETCH FIRST", despite that "FETCH NEXT" is also syntactically correct. During the stage of implementation we agreed with @martint that the clause should be named "FETCH FIRST".

Reply:
Indeed. "fetch" is the name of another feature in the SQL spec: "fetch statement", which is used for interacting with cursors.

Reply:
Agreed. I'll update comments.